src/unitxt/metrics.py

```diff
@@ -2254,6 +2251,53 @@ def compute(
         }
 
 
+class LlamaIndexFaithfulness(LlamaIndexCorrectness):
```

Comment:
There is a lot of duplicated code between the metrics. Can you please create a common base class  (e.g. LlamaIndexMetric), and more the code there?

Reply:
done